Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -531,17 +532,38 @@ def setVolumeNodeProperties(self, volumeNode, loadable):
             #   [2] https://github.com/Slicer/Slicer/blob/3bfa2fc2b310d41c09b7a9e8f8f6c4f43d3bd1e2/Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.h#L172
             #
             try:
-                windowCenter = float(slicer.dicomDatabase.fileValue(file, self.tags["windowCenter"]))
-                windowWidth = float(slicer.dicomDatabase.fileValue(file, self.tags["windowWidth"]))
                 displayNode = volumeNode.GetDisplayNode()
                 if displayNode:
-                    logging.info("Window/level found in DICOM tags (center=" + str(windowCenter) + ", width=" + str(windowWidth) + ") has been applied to volume " + volumeNode.GetName())
-                    displayNode.AddWindowLevelPreset(windowWidth, windowCenter)
+                    # Retrieve the window center and window width values from the DICOM file
+                    windowCenterValues = slicer.dicomDatabase.fileValue(file, self.tags["windowCenter"])
+                    windowWidthValues = slicer.dicomDatabase.fileValue(file, self.tags["windowWidth"])
+
+                    # Split the values by the backslash character to handle multiple values
+                    windowCenterList = windowCenterValues.split("\\")
+                    windowWidthList = windowWidthValues.split("\\")
+
+                    # Ensure both lists have the same length
+                    if len(windowCenterList) != len(windowWidthList):
+                        raise ValueError("The number of window center values does not match the number of window width values")
+
+                    if len(windowCenterList) == 0:
+                        logging.info("No window/level found in DICOM tags. SOPinstanceUID: " + uid)
+
+                    # Iterate over the pairs and add window level presets
+                    for windowCenter, windowWidth in zip(windowCenterList, windowWidthList):
+                        # Convert the values to float
+                        windowCenter = float(windowCenter)
+                        windowWidth = float(windowWidth)
+
+                        # Add the window level preset to the display node
+                        displayNode.AddWindowLevelPreset(windowWidth, windowCenter)
+                        logging.info("Window/level found in DICOM tags (center=" + str(windowCenter) + ", width=" + \
+                                     str(windowWidth) + ") has been applied to volume " + volumeNode.GetName())
                     displayNode.SetWindowLevelFromPreset(0)
                 else:
                     logging.info("No display node: cannot use window/level found in DICOM tags")
             except ValueError:
-                pass  # DICOM tags cannot be parsed to floating point numbers
+                logging.error("The window center/width values cannot be parsed to floating point numbers. SOPinstanceUID: " + uid)
```

Comment:
Let's log what actually happened (it may be that the number of values do not match).
```suggestion
                logging.error(f"Failed to parse window center/width values for volume {volumeNode.GetName()} SOP instance: {uid}. Details: {str(e)}")
```